geode-cq/src/distributedTest/java/org/apache/geode/cache/query/internal/QueryConfigurationServiceConstraintsDistributedTest.java

```diff
@@ -233,41 +167,57 @@ private void createClientCq(String queryString, boolean executeWithInitialResult
    * The operations should succeed, the CQ should fire 'onEvent' and no errors should be logged.
    */
   @Test
-  @Parameters(method = "getRegionTypeOperationsAndCqExecutionType")
-  @TestCaseName("[{index}] {method}(RegionType:{0};Operation:{1},ExecuteWithInitialResults:{2})")
+  @Parameters({
+      "REPLICATE, PUT, true", "REPLICATE, PUT, false",
+      "REPLICATE, CREATE, true", "REPLICATE, CREATE, false",
+      "REPLICATE, REMOVE, true", "REPLICATE, REMOVE, false",
+      "REPLICATE, DESTROY, true", "REPLICATE, DESTROY, false",
+      "REPLICATE, UPDATE, true", "REPLICATE, UPDATE, false",
+      "REPLICATE, REPLACE, true", "REPLICATE, REPLACE, false",
+      "REPLICATE, INVALIDATE, true", "REPLICATE, INVALIDATE, false",
+      "PARTITION, PUT, true", "REPLICATE, PUT, false",
+      "PARTITION, CREATE, true", "REPLICATE, CREATE, false",
+      "PARTITION, REMOVE, true", "REPLICATE, REMOVE, false",
+      "PARTITION, DESTROY, true", "REPLICATE, DESTROY, false",
+      "PARTITION, UPDATE, true", "REPLICATE, UPDATE, false",
+      "PARTITION, REPLACE, true", "REPLICATE, REPLACE, false",
+      "PARTITION, INVALIDATE, true", "REPLICATE, INVALIDATE, false"})
```

Comment:
The set of parameters is duplicated across the test, I believe the previous approach of having a specific method to return the full list of parameters is easier to read. Am I missing something here?.

Reply:
I can restore this. The reason I did this was because the JUnitParams parameter methods are always detected as unused by the IDE, so then you have to mark it with `SuppressWarnings("unused")`.

Reply:
I also prefer to see the parameters with the test method instead of looking for another method. I could try to find a 3rd way to provide the parameters to the tests that isn't redundant.

Reply:
I am don't like the duplication, but I think this way is easier to read. I think it doesn't matter a lot in this case. if the test were getting duplicated a lot or if this was in core (not test) code, I might object more.

Reply:
👍

Reply:
I ended up replacing it with `NamedParameters` method.